scripts/ct/run.sh

```diff
@@ -327,7 +330,7 @@ else
     if [ -z "${REBAR3CT:-}" ]; then
         docker exec -e IS_CI="$IS_CI" \
                     -e PROFILE="$PROFILE" \
-                    -e SUITEGROUP="${SUITEGROUP:-}" \
+                    -e SUITES="${SUITES:-}" \
```

Comment:
this probably shouldn't be touched here.  it's used by `find-suites.sh` in the makefile

Reply:
checked the script, `SUITEGROUP` should be kept here. 👍
add `SUITES` for running sepecific (or just single) SUITE.

Reply:
If you are running specific tests locally, I suggest simply entering a shell inside the container and running them directly.

i.e.

```sh
env DOCKER_CT_RUNNER_IMAGE=ghcr.io/emqx/emqx-builder/5.5-4:1.18.3-27.3.4.2-1-ubuntu24.04 PROFILE=emqx-enterprise scripts/ct/run.sh --app apps/emqx_bridge_datalayers --attach
make apps/emqx_bridge_datalayers-ct
```